docs/pages/guides/display-frames.mdx

```diff
@@ -15,14 +15,14 @@ This guide shows you how to add frames rendering to your next.js + tailwind app
 
 `cd my-project`
 
-`yarn add @farcaster/core frames.js`
+`yarn add @frames.js/render`
```

Comment:
why is this called render? isn't this conflicting with future possibility of having different middleware for rendering images?

Reply:
good point, but i don't think we will call an image rendering middleware `@frames.js/render`

i suppose a better name could be `@frames.js/display`? what do you think @davidfurlong

Reply:
I think you are right since the new package for image rendering would have middleware in its name probably. So I'm fine with this package name.